nilearn/decoding/tests/test_searchlight.py

```diff
@@ -175,3 +179,39 @@ def test_searchlight_group_cross_validation_with_extra_group_variable(
     # run searchlight on list of 3D images
     sl = searchlight.SearchLight(mask_img)
     sl.fit(imgs, y)
+
+
+def test_searchlight_attributes_exist_after_fit():
+    """Test if attributes `process_mask_` and `masked_scores_`
+    exist after fitting.
+    """
+    # Load example dataset
+    haxby = fetch_haxby()
```

Comment:
I don't think we want to run our tests on the example data.

Check the other tests in `nilearn/decoding/tests/test_searchlight.py` to see how they 'mock' data to run the searchlights on.